packages/warriorjs-cli/src/ProfileGenerator.js

```diff
@@ -44,15 +44,32 @@ class ProfileGenerator {
    */
   generateReadmeFile() {
     const template = fs.readFileSync(README_TEMPLATE_FILE_PATH, 'utf8');
-    const data = {
-      getFloorMap,
-      getFloorMapKey,
-      profile: this.profile,
-      level: this.level,
```

Comment:
You just need to change the `level` key to an array of levels, not the entire `data` object. The other three keys remain the same no matter the level.